### PR TITLE
Used signed fetch with Elasticsearch requests

### LINKED ISSUE #22: Use signed fetch with Elasticsearch
See https://financialtimes.slack.com/archives/next-elastic/p1488813543787951 for why, and https://financialtimes.slack.com/archives/next-elastic/p1488814118841422 for how.

I'll look to open a PR.

### PR DESCRIPTION
**Pull request by sjparkinson**
Resolves #22.

### CODE REVIEW

**Comment by sjparkinson** [comment]
The Lambda's need access!

Added in https://github.com/Financial-Times/next-es-interface/pull/786.

File: Makefile [hunk 1]
```diff
@@ -24,11 +24,11 @@ test: verify unit-test local-test
 	@$(DONE)
 
 unit-test:
-	@mocha 'test/**/*.spec.js'
+	@export AWS_SIGNED_FETCH_DISABLE_DNS_RESOLUTION=true; mocha 'test/**/*.spec.js'
```
Comment (bjfletcher) [inline Makefile]: what is this `AWS_SIGNED_FETCH_DISABLE_DNS_RESOLUTION=true` and why is it needed?

Comment (sjparkinson) [inline Makefile]: The source of it is https://github.com/matthew-andrews/signed-aws-es-fetch/blob/master/main.js#L11.

With it set, it doesn't resolve the cname of `next-elastic.ft.com`, which is what makes the mock of fetch possible. Without it set the host is resolved to the Elasticsearch clusters endpoint (EU or US).

I'm not a fan, but that's how it is currently.

Comment (bjfletcher) [inline Makefile]: Ok, can you pretty please move the env var setting into the spec JS...

`process.env.FOO='blah'` in `before`, along with your comment above, and `delete process.env.FOO` in `after`

**Pull request merged**